src/Form/LoginForm.php

```diff
@@ -60,7 +60,7 @@ public function getFormFields()
     public function getFormActions()
     {
         $actions = FieldList::create();
-        $providers = Config::inst()->get($this->authenticator_class, 'providers');
+        $providers = Config::inst()->get($this->getAuthenticatorClass(), 'providers');
```

Comment:
Can you change `composer.json` to require `"silverstripe/framework": "^4.4"`? As that was when this method was first added

Reply:
Ok, sure.